webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -797,19 +797,20 @@ private static TableAndHighlighting createTable(PdfWriter docWriter, Document do
         Font scrambleFont = null;
 
         try {
-            BaseFont courier = BaseFont.createFont(BaseFont.COURIER, BaseFont.CP1252, BaseFont.EMBEDDED);
+            // load the font from the resource folder. This loads the regular version, but can be changed to light, bold, etc if people prefer that
```

Comment:
nit: For consistency's sake, lets do full sentences (capital letter at the start, and periods at the end).